build.config.json

```diff
@@ -1,6 +1,7 @@
 {
     "appId": "com.electron.hugin",
-    "productName": "Hugin Messenger",
+    "productName": "HuginMessenger",
```

Comment:
Changing the productName so the target name will be the same all the time since versions are not neccessary since we have our update functionality.